tests/app/celery/test_tasks.py

```diff
@@ -1103,7 +1103,7 @@ def test_process_rows_works_without_key_type(
                 "to": "recip",
                 "row_number": "row_num",
                 "personalisation": {"foo": "bar"},
-                "queue": QueueNames.SEND_SMS_MEDIUM if template_type == SMS_TYPE else "send-{}-tasks".format(template_type),
+                "queue": QueueNames.SEND_SMS_MEDIUM if template_type == SMS_TYPE else QueueNames.SEND_EMAIL_MEDIUM,
```

Comment:
the SMS ones are being set to `QueueNames.SEND_SMS_MEDIUM` as before. Previously the `else` really should have been just `else "send-email-tasks"` since there's no letter type in the test parameters.

Reply:
oh I read that change incorrectly, it's all orthogonal now and good